frontend/public/components/pod.tsx

```diff
@@ -29,17 +32,61 @@ import {
   ScrollToTopOnMount,
   SectionHeading,
   Timestamp,
+  formatBytesAsMiB,
+  formatCores,
   humanizeBinaryBytes,
   humanizeCpuCores,
   navFactory,
   pluralize,
   units,
 } from './utils';
+import { fromNow } from './utils/datetime';
 import { PodLogs } from './pod-logs';
-import { requirePrometheus, Area } from './graphs';
+import {
+  Area,
+  PROMETHEUS_BASE_PATH,
+  PROMETHEUS_TENANCY_BASE_PATH,
+  requirePrometheus,
+} from './graphs';
 import { CamelCaseWrap } from './utils/camel-case-wrap';
 import { VolumesTable } from './volumes-table';
 
+// Only request metrics if the device's screen width is larger than the
+// breakpoint where metrics are visible.
+const showMetrics =
+  PROMETHEUS_BASE_PATH && PROMETHEUS_TENANCY_BASE_PATH && window.screen.width >= 992;
```

Comment:
```suggestion
  PROMETHEUS_BASE_PATH && PROMETHEUS_TENANCY_BASE_PATH && window.screen.width >= 1200;
```
Since the cols aren't visibile until [xl](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/sass-utilities/scss-variables.scss#L161)